cli/src/scripts/rate.ts

```diff
@@ -49,7 +53,12 @@ export default ({ command }: RootCommand): Argv<unknown> =>
           choices: ["A", "B"],
           description: "assessed quality",
         })
-        .demandOption(["key", "quality"]),
+        .option("reason", {
+          alias: "r",
+          type: "string",
+          describe: "reason for assessment",
+        })
+        .demandOption(["key", "quality", "reason"]),
```

Comment:
Worth grouping quality and reason?

Reply:
Good idea. It would be nice to group connection details as well (and make sure that they work as env vars). I'll create a separate issue to clean up args before they get out of hand.